SU2_CFD/include/solvers/CFVMFlowSolverBase.hpp

```diff
@@ -29,15 +29,20 @@
 #include "../../../Common/include/omp_structure.hpp"
 #include "../../../Common/include/toolboxes/geometry_toolbox.hpp"
 #include "CSolver.hpp"
+#include "../variables/CNEMOEulerVariable.hpp"
+
+
```

Comment:
move this include to CNEMOEulerSolver (before the include of FVMBase) otherwise when you change CNEMOEulerVariable all derived solvers of FVMBase are forced to recompile.

Reply:
Nevermind what I said, it would not work, instead delete the include from this file and do a forward declaration of CNEMOEulerVariable (`class CNEMOEulerVariable;`).